pode.build.ps1

```diff
@@ -212,17 +241,39 @@ task Build BuildDeps, {
 #>
 
 # Synopsis: Creates a Zip of the Module
-task 7Zip -If (Test-PodeBuildIsWindows) PackDeps, StampVersion, {
+Task 7Zip -If (Test-PodeBuildIsWindows) PackDeps, StampVersion, {
     exec { & 7z -tzip a $Version-Binaries.zip ./pkg/* }
 }, PrintChecksum
 
+
+# Synopsis: Creates a Zip of the Module
+Task Compress StampVersion, {
+    $path = './deliverable'
+    if (Test-Path $path) {
+        Remove-Item -Path $path -Recurse -Force | Out-Null
+    }
+    # create the pkg dir
+    New-Item -Path $path -ItemType Directory -Force | Out-Null
+    Compress-Archive -Path './pkg' -DestinationPath "$path/$Version-Binaries.zip"
```

Comment:
`./pkg` on `Compress-Archive` will need to be `./pkg/*`, otherwise the zip created will contain the pkg folder as well.